test/e2e/join_and_leave_test.go

```diff
@@ -95,9 +100,86 @@ var _ = Describe("Test member cluster join and leave flow", Ordered, Serial, fun
 			}
 		})
 
-		It("Should be able to unjoin a cluster with crp still running", func() {
-			By("remove all the clusters without deleting the CRP")
+		It("create a dummy internalServiceExport in the reserved member namespace", func() {
```

Comment:
Just a nit: it's my personal opinion, but it might be clearer if we refactor this into two separate test cases?

Reply:
prefer to keep the current one to make the case complex to ensure it works as expected :)